packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/utils/FlowableTests.kt

```diff
@@ -49,6 +49,24 @@ interface FlowableTests {
     @Test
     fun closingRealmDoesNotCancelFlows()
 
+    @Test
+    fun keyPath_topLevelProperty()
+
+    @Test
+    fun keyPath_nestedProperty()
+
+    @Test
+    fun keyPath_propertyBelowDefaultLimit()
```

Comment:
I guess we should/could also have a test that asserts that we don't receive events below the default level if we don't register a key path. Just to ensure that we track changed behavior if it ever were to change.